src/data/plants/toxic.json

```diff
@@ -92,19 +105,48 @@
       "lat": "strelitzia reginae"
     },
     "slug": "strelicja",
-    "thumbnailID": "866733515",
-    "danger": 4
+    "imageID": [
+      "866733515",
+      "4814038355",
+      "2524917491",
+      "17563691411",
+      "8160264472",
+      "5624276430"
+    ],
+    "danger": 1,
+    "symptoms": "łagodne mdłości, wymioty, ospałość",
+    "note": "głównie owoce i nasiona",
+    "source": [
+      "https://www.aspca.org/pet-care/animal-poison-control/toxic-and-non-toxic-plants/bird-paradise-flower",
+      "https://plants.ces.ncsu.edu/plants/strelitzia-reginae/",
+      "https://pets.thenest.com/birds-paradise-cats-11340.html"
+    ]
   },
   {
     "name": {
       "pl": ["aloes"],
-      "en": ["aloe vera"],
+      "en": ["aloe vera", "Barbados aloe", "the want of heaven"],
       "lat": "aloe vera"
     },
     "queryImage": "aloe vera leaves",
     "slug": "aloes",
-    "thumbnailID": "50658583448",
-    "danger": 4
+    "imageID": [
+      "4626376701",
+      "8737144945",
+      "13834577",
+      "4626377063",
+      "8463896997",
+      "4743154534"
+    ],
+    "danger": 2,
+    "symptoms": "wymioty, ospałość, biegunka",
+    "note": "żel aloesowy jest bezpieczny",
+    "source": [
```

Comment:
some plants that have 100% data (and multiple images): frezja, fuksja, monstera, aloes, strelicja